### ISSUE TITLE
Accuracy Issue

### ISSUE THREAD

**Issue by 1541734425** [serengil/deepface#261]
Hi, serengil. I use your code to test your model on the LFW. But the accuracy is not what you mentioned in your readme.md.   Facenet and ArcFace got excellent performance, but the others got very low accuracy. Have you tested all models? And is the threshold right? Could you give me some advice about this problem?

**Comment by serengil** [comment]
No, I didn't tested those models. Those are the values reference studies declared. Because I didn't train them. I used pre-trained models.

BTW, I alread mentioned them in my posts:

Blog: https://sefiks.com/2020/08/27/labeled-faces-in-the-wild-for-face-recognition/ (Dlib got 92% here)

Vlog: https://youtu.be/9fT-9TFjeDQ (ArcFace got 93% here)

Reference studies might use sup-samples in LFW or this is because of they are using different procedures for detection and alignment. The first one is more possible.

**Comment by 1541734425** [comment]
Thanks for your advice. I saw the video and the blog, which helped me a lot. But I still question the detail about the value of threshold within the function findThreshold in the code commons/distance.py. You know, the threshold is significant for us to verify image pairs, as we determine whether the image pairs are the same person. Could you tell me why you set the value like your function shows or share me the channels you refer to. Thanks again. Looking forward to your reply.

Below is your function(findThreshold) I mentioned.

'''python
def findThreshold(model_name, distance_metric):

	base_threshold = {'cosine': 0.40, 'euclidean': 0.55, 'euclidean_l2': 0.75}

	thresholds = {
		'VGG-Face': {'cosine': 0.40, 'euclidean': 0.55, 'euclidean_l2': 0.75},
		'OpenFace': {'cosine': 0.10, 'euclidean': 0.55, 'euclidean_l2': 0.55},
		'Facenet':  {'cosine': 0.40, 'euclidean': 10, 'euclidean_l2': 0.80},
		'DeepFace': {'cosine': 0.23, 'euclidean': 64, 'euclidean_l2': 0.64},
		'DeepID': 	{'cosine': 0.015, 'euclidean': 45, 'euclidean_l2': 0.17},
		'Dlib': 	{'cosine': 0.07, 'euclidean': 0.6, 'euclidean_l2': 0.6},
		'ArcFace':  {'cosine': 0.6871912959056619, 'euclidean': 4.1591468986978075, 'euclidean_l2': 1.1315718048269017}
		}

	threshold = thresholds.get(model_name, base_threshold).get(distance_metric, 0.4)

	return threshold
'''

**Comment by serengil** [comment]
Here, I explained how to determine those thresholds: https://sefiks.com/2020/05/22/fine-tuning-the-threshold-in-face-recognition/

I hope this helps you

**Comment by sri9s** [comment]
> 

@serengil As you see in the threshold list here VGG-Face and euclidean_l2 was 0.75 but now when I run it shows 
`{"distance":0.7931165562316705,"max_threshold_to_verify":0.86,"similarity_metric":"euclidean_l2","verified":true}`
threshold is changed to 0.86.

Has there been an update in the model's thresholds?
